### ISSUE TITLE
UA 53: Steve Marino Photographs: Round 3 (Priority 12)

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#329]
### Collection Information

_This information is consistent with Round 1._

Collection description:

The collection consists of a series of photographs of the UC Santa Cruz campus created by alumnus Stephen Marino beginning in 2014.

### Objects

Type/format and dimensions of objects (within each collection or series) & quantity:  Born-digital .tif images

### Rights Information

_This information is consistent with Round 1._

Processing archivist(s):

Section completed by: RJ

## Section 2: Metadata Assessment

Metadata source(s): Donor-provided spreadsheets

Metadata properties to be included:

_This information is consistent with Round 1._

Metadata specialist(s): RJ

Section completed by: RJ

## Section 3: Digitization/Technical Processing Assessment

File format(s) for preservation masters:

File naming conventions:

File directory on Artifacts2:

## Section 4: File assessment and processing
(This section is commonly used for previously digitized or outsourced digitization projects)

- [ ] Master files located and organized on Artifacts

- [ ] File quality assessed and standards documented on the digital assessment worksheet

- [ ] Files processed if needed

- [ ] Filenames matched to metadata spreadsheet

- [ ] Files saved on dams_ingest

## Section 5: DAMS Development Assessment

Development work required?: Y/N   (if Y →)
    
Brief description of new feature:
    

Subject Matter Expert(s):
    
Developer(s):

## Section 6: Display Assessment

Modifcations necessary for collection landing page:

Thumbnail Image for DAMS home page:

Brief Description:

Customizations, e.g., new format icons:

## Section 7: Metadata & File Locations

Metadata spreadsheet:
Files:

## Actions & Acceptance Criteria 
_(workflow specific to each collection, i.e., the steps required to prepare it for ingest)_

- [ ]  Collection Information and planning/assessment sections (1-7) completed
- [ ]  Metadata spreadsheet prepared and finalized by RJ
- [ ]  Files assessed and prepared for ingest
- [ ]  Customizations for display are discussed and documented

**Comment by rmjaffe** [comment]
Round #3 is on hold until the donor has finished work on providing descriptive metadata.

**Comment by rmjaffe** [comment]
Update: Metadata has been received from donor.

**Comment by tmariamora** [comment]
@rmjaffe Should I move this card back to the Preparation column since you're waiting on me?

**Comment by rmjaffe** [comment]
Update the metadata for the Round 3 photos (2018) have been appended to the combined spreadsheet for Rounds 1 (2014-2016) and 2 (2017).  These will move forward as a single spreadsheet and single ingest.
